www/docs/server/non-json-content-types.md

```diff
@@ -0,0 +1,104 @@
+---
+id: non-json-content-types
+title: Non-JSON Content Types
+sidebar_label: Non-JSON Inputs (FormData, File, Blob)
+slug: /server/non-json-content-types
+---
+
+In addition to JSON-serializable data, tRPC can use FormData, File, and other Binary types as procedure inputs
+
+## Client Setup
+
+:::warning
```

Comment:
warning seems a bit hostile

```suggestion
:::info
```
?

Reply:
I like warning because it's a common foot-gun but happy to change